### PR TITLE
fix related objects field names

### PR DESCRIPTION
**Pull request by cleberzavadniak**
Thus the same payload got through a GET can be used to PUT or PATCH the object.

### CODE REVIEW

**Review by erickravaneli — DISMISSED** [review]
.

File: powerlibs/django/restless/models.py [hunk 1]
```diff
@@ -82,26 +82,24 @@ def serialize_model(obj, fields=None, include=None, exclude=None, fixup=None):
...
+    if exclude:
         fields = [f for f in fields if f not in exclude]
 
-    if include is not None:
-        for i in include:
-            if isinstance(i, tuple) or (isinstance(i, six.string_types)):
-                fields.append(i)
+    include = include or []
+    for i in include:
+        if isinstance(i, tuple) or (isinstance(i, six.string_types)):
```
Comment (adolfosabino) [inline powerlibs/django/restless/models.py]: Pode ser 
isinstance(i, (tuple, six.string_types))

Comment (cleberzavadniak) [reply powerlibs/django/restless/models.py]: Yeap. Sorry for that. Fixing.

**Review by adolfosabino — APPROVED** [review]
:+1:

**Review by laylacodogno — APPROVED** [review]
👍

**Pull request merged**